src/index.ts

```diff
@@ -74,6 +74,14 @@ interface ActualDefinitionNode {
 const undefinedVariableConst = "undefined_variable"
 const isVariableDropinConst = "_____isVariableDropinConst"
 
+Object.defineProperty(Array.prototype, "flatMap", {
+    value: function(callback, thisArg) {
+        return this.map(callback, thisArg).reduce((a, b) => a.concat(b), [])
+    },
+    configurable: true,
+    writable: true,
+})
+
```

Comment:
`.Array.flatMap()` is only available in Node.js v12 and above - hence why I've added a polyfill function

Reply:
Here I have to give my veto - this pattern is called prototype pollution and gets flagged up by npm if you overwrite or attach things to existing things. Here, you would override the existing Array.prototype.flatMap if it was given.

Could you please instead use a regular function?